Source/SvgFontManager.cs

```diff
@@ -12,11 +12,19 @@ namespace Svg
     /// When a font is requested in the render process, if the font is not found as an embedded SvgFont, the render
     /// process will SvgFontManager.FindFont method.
     /// </summary>
-
     public static class SvgFontManager
     {
+        private static readonly string[][] LocalizedFamilyNames = new string[][]
+        {
+            // Japanese
```

Comment:
Add other localization if needed.

Reply:
Wouldn't it make sense to allow to add more localized names via the API? Collecting them all here will be a bit cumbersome. That being said, I agree that this certainly is an improvement :)